frontend/javascripts/oxalis/view/layouting/flex_layout_wrapper.js

```diff
@@ -105,6 +105,7 @@ class FlexLayoutWrapper extends React.PureComponent<Props, State> {
       }),
     );
     this.unbindListeners.push(this.attachMaximizeListener());
+    this.unbindListeners.push(this.attachToggleBorderListener());
```

Comment:
I think, it would be better to merge these two functions (e.g., `attachKeyboardShortcuts`), as there is no need for having two separate `InputKeyboard` instances. The instantiation would look somewhat like this:

```
    const keyboardNoLoop = new InputKeyboardNoLoop(
      {
         k: () => this.toggleBorder("left"),
         l: () => this.toggleBorder("right") },
         ".": toggleMaximize
       }
      { supportInputElements: false },
    );
```